src/System.Drawing.Common/tests/Drawing2D/GraphicsPathTest.cs

```diff
@@ -0,0 +1,3294 @@
+                new GraphicsPath().AddString("mono", null, 0, 10, new Point(10, 10), StringFormat.GenericDefault));
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void Transform_Success()
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            using (Matrix matrix = new Matrix(1f, 1f, 2f, 2f, 3f, 3f))
+            {
+                gp.AddRectangle(new Rectangle(1, 1, 2, 2));
+                AssertRectangle(gp);
+                gp.Transform(matrix);
+                Assert.Equal(new float[] { 1f, 1f, 2f, 2f, 3f, 3f }, matrix.Elements);
+
+                RectangleF rect = gp.GetBounds();
+                AssertEqual(6f, rect.X, Delta);
+                AssertEqual(6f, rect.Y, Delta);
+                AssertEqual(6f, rect.Width, Delta);
+                AssertEqual(6f, rect.Height, Delta);
+
+                AssertEqual(6f, gp.PathData.Points[0].X, Delta);
+                AssertEqual(6f, gp.PathData.Points[0].Y, Delta);
+                Assert.Equal(0, gp.PathData.Types[0]);
+                AssertEqual(8f, gp.PathData.Points[1].X, Delta);
+                AssertEqual(8f, gp.PathData.Points[1].Y, Delta);
+                Assert.Equal(1, gp.PathTypes[1]);
+                AssertEqual(12f, gp.PathData.Points[2].X, Delta);
+                AssertEqual(12f, gp.PathData.Points[2].Y, Delta);
+                Assert.Equal(1, gp.PathData.Types[2]);
+                AssertEqual(10f, gp.PathData.Points[3].X, Delta);
+                AssertEqual(10f, gp.PathData.Points[3].Y, Delta);
+                Assert.Equal(129, gp.PathTypes[3]);
+            }
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void Transform_PathEmpty_Success()
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            using (Matrix matrix = new Matrix(1f, 1f, 2f, 2f, 3f, 3f))
+            {
+                gp.Transform(matrix);
+                Assert.Equal(new float[] { 1f, 1f, 2f, 2f, 3f, 3f }, matrix.Elements);
+                AssertEmptyGrahicsPath(gp);
+            }
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void Transform_MatrixNull_ThrowsArgumentNullException()
+        {
+            Assert.Throws<ArgumentNullException>(() => new GraphicsPath().Transform(null));
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void GetBounds_PathEmpty_ReturnsExpected()
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            {
+                AssertRectangleBounds(gp.GetBounds(), 0f, 0f, 0f, 0f);
+            }
+        }
+
+        private static IEnumerable<object[]> GetBounds_TestData()
+        {
+            yield 
```

Comment:
I think dan already mentioned it but can you check if this works on Windows. If it doesn't please update the test and remove the skip attribute.